source/dub/generators/build.d

```diff
@@ -52,6 +52,7 @@ string computeBuildName(string config, in GeneratorSettings settings, const stri
 	{
 		import std.path:stripExtension, baseName;
 		string recipeName = settings.recipeName.baseName.stripExtension;
+		addHash(settings.recipeName);
```

Comment:
this doesn't re-evaluate the hashstr, you need to call addHash above

Reply:
Fixed, sorry for the dumb mistake :(